tests/screenshot.spec.ts

```diff
@@ -135,6 +134,35 @@ browserTest.describe('element screenshot', () => {
     await context.close();
   });
 
+  browserTest('element screenshot should include content outside viewport', async ({ browserName, headless, platform, contextFactory }) => {
```

Comment:
I liked previous naming that included vh more, also was nice to have two separate tests.